projects/fsastorefrontlib/src/cms-components/checkout/components/application-confirmation/application-confirmation.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { DomSanitizer } from '@angular/platform-browser';
+import { RoutingService } from '@spartacus/core';
+import { ApplicationConfirmationComponent } from './application-confirmation.component';
+
+const mockImageSrc =
```

Comment:
remove this real image, put anything dummy. After all, you won't check after if image is rendered.
For example 'data:/image/svg+xml;base65, sampleImageCode';

Reply:
Real image replaced with a mocked image